packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -5,777 +5,413 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+   * @notice Returns the pending withdrawals from unlocked gold for an account.
+   * @param account The address of the account.
+   * @return The value and timestamp for each pending withdrawal.
    */
-  function updateNotifiedDeposit(
-    Account storage account,
-    uint256 value,
-    uint256 availabilityTime
+  function getPendingWithdrawals(
+    address account
   )
-    private
+    public
+    view
+    returns (uint256[] memory, uint256[] memory)
   {
-    Commitment storage notified = account.commitments.notified[availabilityTime];
-    require(value != notified.value);
-    if (notified.value == 0) {
-      notified.index = uint128(account.commitments.availabilityTimes.length);
-      notified.value = uint128(value);
-      account.commitments.availabilityTimes.push(availabilityTime);
-      account.weight = account.weight.add(notified.value);
-      totalWeight = totalWeight.add(notified.value);
-    } else if (value == 0) {
-      account.weight = account.weight.sub(notified.value);
-      totalWeight = totalWeight.sub(notified.value);
-      deleteCommitment(notified, account.commitments, CommitmentType.Notified);
-    } else {
-      uint256 difference;
-      if (value >= notified.value) {
-        difference = value.sub(notified.value);
-        account.weight = account.weight.add(difference);
-        totalWeight = totalWeight.add(difference);
-      } else {
-        difference = uint256(notified.value).sub(value);
-        account.weight = account.weight.sub(difference);
-        totalWeight = totalWeight.sub(difference);
-      }
-
-      notified.value = uint128(value);
+    require(isAccount(account));
+    uint256 length = accounts[account].balances.pendingWithdrawals.length;
+    uint256[] memory values = new uint256[](length);
+    uint256[] memory timestamps = new uint256[](length);
+    for (uint256 i = 0; i < length; i++) {
+      PendingWithdrawal memory pendingWithdrawal = (
+        accounts[account].balances.pendingWithdrawals[i]
+      );
+      values[i] = pendingWithdrawal.value;
+      timestamps[i] = pendingWithdrawal.timestamp;
     }
+    return (values, timestamps);
   }
 
   /**
-   * @notice Deletes a commitment from an account.
-   * @param _commitment The commitment to delete.
-   * @param commitments The struct containing the account's commitments.
-   * @param commitmentType Whether the deleted commitment is locked or notified.
+   * @notice Authorizes voting or validating power of `msg.sender`'s account to another address.
+   * @param current The address to authorize.
+   * @param previous The previous authorized address.
+   * @param v The recovery id of the incoming ECDSA signature.
+   * @param r Output value r of the ECDSA signature.
+   * @param s Output value s of the ECDSA signature.
+   * @dev Fails if the address is already authorized or is an account.
+   * @dev v, r, s constitute `authorize`'s signature on `msg.sender`.
```

Comment:
There's no variable called `authorize`. Presume this should be `current`?

Reply:
Danger of search and replace